core/src/main/java/tc/oc/pgm/spawner/SpawnerDefinition.java

```diff
@@ -19,15 +19,16 @@ public class SpawnerDefinition implements FeatureDefinition {
   public final int numericID;
 
   public SpawnerDefinition(
+      int numericID,
```

Comment:
```suggestion
      String id,
```

Make SpawnerDefinition extend SelfIdentifyingFeatureDefinition, and pass this id as param to super constructor